src/main/java/racingcar/step3/view/InputView.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.step3.view;
+
+import racingcar.step3.common.Consts;
+import racingcar.step3.common.Scan;
+
+import java.util.Scanner;
+
+/**
+ * Created : 2020-11-02 오전 8:25
+ * Developer : Seo
+ */
+public class InputView {
+    private int cars;
+    private int turns;
+    Scanner scanner;
+
+    public void setCars() {
+        this.scanner = Scan.getInstance();
+        System.out.println(Consts.HOW_MANY_CARS);
+        this.cars = scanner.nextInt();
```

Comment:
cars를 상태값으로 두지 않고
바로 return 해줘도 좋을거 같아요! 😄 
```suggestion
        return scanner.nextInt();
```